docs/future-of-typeorm.md

```diff
@@ -0,0 +1,52 @@
+# The Future of TypeORM
+
+> By [David Hoeck (@dlhck)](https://github.com/dlhck) and [Michael Bromley (@michaelbromley)](https://github.com/michaelbromley)
+
+
+TypeORM is one of the most high-performance, feature-rich, and battle-tested ORMs in the Node.js ecosystem, relied upon by hundreds of thousands of projects and companies worldwide. With nearly 2 million downloads each week, it powers countless applications as a critical dependency. However, over the past few years, maintenance has slowed significantly, leading to growing uncertainty about the project’s future among its dedicated community.
+
+We’re thrilled to announce that Michael Bromley and David Hoeck, under the umbrella of our parent company [Elevantiq](https://elevantiq.com/), are
+stepping up to lead TypeORM into its next chapter. At Elevantiq, where we
+specialize in enterprise digital commerce solutions, TypeORM is a critical dependency—not
+only for [Vendure](https://vendure.io/), our flagship open-source project, but also for many of our other solutions.
+With our reliance on TypeORM and the growing needs of its vibrant community, we saw an
+opportunity to contribute back and ensure the project remains active, maintained, and secure.
+
+After discussions with TypeORM’s original maintainers, Umed and Dmitry, we’ve reached an agreement to take on the project’s maintenance, inspired by successful, community-centric open-source projects. One standout model is the Tauri project, a self-governing open-source initiative co-founded by Daniel, who we already collaborate with through Michael and Vendure. Our discussions with Tauri have set the foundation for the organizational structure we envision.
+
+## Our Vision for TypeORM’s Future
+
+To ensure long-term stability and governance, we plan to establish a non-profit
+foundation for TypeORM, likely under the [Commons Conservancy](https://commonsconservancy.org/) in the Netherlands.
+This foundation will be led by a board of seven members: Michael and David,
+along with five additional board members dedicated to the project’s success.
+The board will work closely with a **Working Group** comprising companies, contributors,
+and other community members who heavily rely on TypeORM. This collaborative setup will help guide
+strategic decisions that align with the needs and goals of TypeORM’s ecosystem.
+
+## Organizational Structure
+
+To drive TypeORM forward, we’ll introduce three core domains:
+
+- **Development**: Led by core developers with expertise across various database engines and adapters, this team will focus on TypeORM’s architecture and ongoing technical maintenance and advancements.
+- **Operations**: Handling day-to-day needs, from documentation to sponsorship accounting, this team will keep the project running smoothly.
+- **Community**: Dedicated to engaging, moderating, and supporting our growing community, this team will foster collaboration through platforms like Discord.
+
+TODO: Add mockup of the new structure
```

Comment:
todo needs doing :)

Reply:
Removed it, as Mermaid is not supported and not sure if images are currently supported well in the docs.